internal/pkg/skuba/deployments/ssh/gangway.go

```diff
@@ -66,3 +70,35 @@ func gangwayDeploy(t *Target, data interface{}) error {
 	_, _, err = t.ssh("kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f /tmp/gangway.d")
 	return err
 }
+
+const (
+	gangwaySecretName = "oidc-gangway-cert"
```

Comment:
I mean there exist const variable in https://github.com/SUSE/skuba/blob/master/internal/pkg/skuba/gangway/gangway.go#L43, re-use it.